.github/workflows/docs-validate-nav-build.yml

```diff
@@ -39,9 +46,3 @@ jobs:
         working-directory: docs
         run: |
           diff -q docs/nav.js docs/nav.js.original || (echo "Generated nav.js differs from committed version. Run 'npm run build' and commit the updated file." && exit 1)
-
-      - name: Restore original nav.js
-        working-directory: docs
-        if: success() || failure()
-        run: |
-          mv docs/nav.js.original docs/nav.js
```

Comment:
I removed this step because it would fail if the "Check that all files are listed in nav.js" step failed (because we'd never get to the step that creates `nav.js.original`), and more importantly this step seems completely unnecessary? The checkout is ephemeral.